warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -527,13 +524,26 @@ public void run() {
                     log.error("Error: " + e.getMessage(), e);
                 }
             }
+        } catch (IOException e) {
+            log.error("Error: " + e.getMessage(), e);
         } finally {
             log.info("Shutting down executor service");
             executor.shutdown();
         }
         log.info("Bulk map file loader shutting down.");
     }
     
+    protected void cleanJobDirectoriesOnStartup() throws IOException {
+        Path[] cleanupDirectories = getJobDirectories(destHdfs, new Path(workDir, jobDirPattern + '/' + CLEANUP_FILE_MARKER));
+        for (int i = 0; i < cleanupDirectories.length; i++) {
+            
+            markSourceFilesLoaded(cleanupDirectories[i]);
+            getFileSystem(destHdfs).delete(cleanupDirectories[i], true);
```

Comment:
Will this throw an exception is some other process already deleted the directory?

Reply:
Ah, you're right.  I'll adjust that try/catch